### ISSUE TITLE
add statsd metrics

### ISSUE THREAD

**Issue by Dieterbe** [raintank/graphite-api#11]
we know how long things take in our finder plugin,
but we don't know how long it takes graphite-api to satisfy an entire request, how long it takes to do whatever it needs to do: runtime consolidation, function processing, etc
as far as i can tell, we currently have no idea where graphite-api is spending most of its time, we think it's runtime consolidation and the like but we don't have the graphs to back that up. right, @woodsaj ?

**Comment by woodsaj** [comment]
correct.  Graphite-api has support for statsd, but does not use it itself.  Pretty sure it was add to support your influxdb finder.
